datadog_checks_dev/datadog_checks/dev/plugin/pytest.py

```diff
@@ -166,9 +166,22 @@ def run_check(config=None, **kwargs):
 
         _, _, collector_output = result.stdout.partition(AGENT_COLLECTOR_SEPARATOR)
         collector_output = collector_output.strip()
-        if not collector_output.endswith(']'):
+
+        if not collector_output.endswith('} ]'):
```

Comment:
I think this will break every non-JMX check. Can you try one?

Reply:
Works with network, process, and apache.

It's not much different than the previous line, the key problem was some extra text still ended with a bracket so it wasn't getting stripped appropriately.

Reply:
It actually needs to be expanded - see latest updates on #5261 

Assuming that PR gets approved first, will update this PR with that version or resolve via merge.

Reply:
Reverted the change - had extraneous output in my dev environment which has been fixed.